Cargo.toml

```diff
@@ -21,7 +21,7 @@ name = "duckdb"
 members = ["libduckdb-sys", "duckdb-loadable-macros"]
 
 [features]
-default = []
+default = ["bundled","vtab","vtab-arrow"]
```

Comment:
don't suggest to  change the default features

Reply:
[    pub fn append_record_batch(&mut self, record_batch: RecordBatch) -> Result<()> {](https://github.com/web3creator/duckdb-rs/blob/2078ebcabf7c44e785da49471b40761e0b073c5e/src/appender.rs#L102) There are some features that will be relied upon here

Reply:
You can move the change in appender.rs into appender_arrow.rs

and change lib.rs with something like:

```rust
#[cfg(feature = "vtab-arrow")]
mod appender_arrow;
```